app/src/main/res/layout/tab_tray_item.xml

```diff
@@ -81,7 +80,6 @@
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_alignParentTop="true"
-        android:layout_marginTop="4dp"
```

Comment:
I think we would want a UX review for this change since we are getting rid of the 4dp margin top on the tab URL that we can see between the title of a tab and the URL.

Reply:
Thanks for the ping @gabrielluong ! I can't reproduce the original issue in the latest nightly so I'm curious whether this also applies to history or bookmark listings?

Reply:
This margin top change only applies to the tabs tray.